### PR TITLE
apps: Make corresponding MCUmgr changes

### PR DESCRIPTION
**Pull request by vrahane**
These changes are corresponding to the changes made to mynewt-core and mynewt-mcumgr.
Please wait for the following pull requests to be merged before merging this PR:
1. https://github.com/apache/mynewt-mcumgr/pull/27
2. https://github.com/apache/mynewt-core/pull/1917

### CODE REVIEW

File: apps/blemesh/syscfg.yml [hunk 1]
```diff
@@ -26,8 +26,8 @@ syscfg.vals:
     # Default task settings
     OS_MAIN_STACK_SIZE: 768
 
-    # Newtmgr is not supported in this app, so disable newtmgr-over-shell.
-    SHELL_NEWTMGR: 0
+    # SMP is not supported in this app, so disable smp-over-shell.
```
Comment (sjanc) [inline apps/blemesh/syscfg.yml]: I'd emphasize here (and in other places too) that this is "Mcumgr SMP" and not "BLE SMP (Security Manager Protocol)"

Comment (rymanluk) [reply apps/blemesh/syscfg.yml]: agree with @sjanc, also would be good to explain the abbreviation SMP in Mcumgr context

Comment (vrahane) [reply apps/blemesh/syscfg.yml]: Sounds good, I’ll add some more context

**Comment by vrahane** [comment]
@sjanc and @rymanluk can you guys please approve it, made changes as requested ?

**Pull request merged**